workbench-service/semantic_workbench_service/start.py

```diff
@@ -33,7 +34,7 @@ def main():
         help="host IP to run service on",
     )
     parse_args.add_argument(
-        "--port", dest="port", type=int, default=settings.service.port, help="port to run service on"
+        "--port", dest="port", type=int, default=int(os.getenv("PORT") or settings.service.port), help="port to run service on"
```

Comment:
this change is unnecessary. the `settings.service.port` already reads from the `PORT` env var. :)

Reply:
can you revert this change?